core/src/test/java/com/github/stefvanschie/quickskript/core/psi/TestClassBase.java

```diff
@@ -36,11 +36,10 @@
             for (File file : files) {
                 result.add(new Skript(SkriptFile.getName(file), SkriptFile.load(file)));
             }
+            return result;
         } catch (IOException e) {
             throw new RuntimeException(e);
         }
-
-        return result;
     }
```

Comment:
Could you explain your reasoning for moving the return statement inside the try/catch? I don't really have anything against it, I'm just curious if there's any reason why the return statement being inside the try/catch would be better.

Reply:
The IntelliJ inspection "return separated from computation of value" flagged it. It doesn't really matter in this case, but the inspection does help create more readable code sometimes - you don't have to check that many branches when you return as soon as possible instead of relying on a return keyword that is placed a few lines below.

Reply:
Okay, thanks for explaining.